lib/chef/knife/ec2_server_create.rb

```diff
@@ -67,10 +67,18 @@ class Ec2ServerCreate < Knife
         :proc => Proc.new { |groups| groups.split(',') }
 
       option :security_group_ids,
-        :short => "-g 'X,Y,Z'",
-        :long => "--security-group-ids 'X,Y,Z'",
-        :description => "The security group ids for this server; required when using VPC,Please provide values in format --security-group-ids 'X,Y,Z'",
-        :proc => Proc.new { |security_group_ids| security_group_ids.split(',') }
+        :short => "-g SECURITY_GROUP_IDS",
+        :long => "--security-group-ids",
```

Comment:
I think since its deprecated, it should still "work" until it is removed so I'd keep current validation and functionality in tact with just the additional deprecation notice as mentioned above.